requirements/ci/py36.yml

```diff
@@ -46,3 +46,6 @@ dependencies:
   - sphinx-copybutton
   - sphinx-gallery
   - sphinx_rtd_theme
+  - pip
+  - pip:
+    - sphinxcontrib-napoleon
```

Comment:
@tkknight Could you also add `sphinxcontrib-napolean` to the `requirements/docs.txt` file - this is part of the requirements for a `pip` install of `iris`.